src/components/appNavigation/AppNavigation.js

```diff
@@ -24,6 +26,10 @@ const routes = {
     screen: Profile,
     display: false,
   },
+  LoginRedirect: {
```

Comment:
have you verified user is redirected to this screen also when
- user is logged out
- user opens the login with gooddollar link on some website
- user logs in into wallet
??
- user is redirected to LoginRedirect

Reply:
@harjaapdhillon16 waiting response

Reply:
The flow is working exactly like this perfectly